lib/fast_delete_from_db_and_es.rb

```diff
@@ -8,7 +8,7 @@ def fast_delete(ids)
       return if ids.blank?
 
       "Elastic#{name}".constantize.delete(ids)
-      delete_all(id: ids)
+      where(:id => ids).delete_all unless (where(:id => ids) == nil)
```

Comment:
A couple of changes:
1) use Ruby 1.9 hash syntax: `where(id: ids).delete_all`
2) The `unless` clause isn't necessary, because if no records are found, then no records will be deleted.